backend/src/main/java/heartbeat/service/report/CSVFileGenerator.java

```diff
@@ -444,9 +444,14 @@ private List<String[]> getRowsFormClassification(Classification classificationLi
 		List<String[]> rows = new ArrayList<>();
 		String fieldName = String.valueOf((classificationList.getFieldName()));
 		List<ClassificationInfo> pairList = classificationList.getClassificationInfos();
-		pairList.forEach(nameValuePair -> rows
-			.add(new String[] { "Classifications", fieldName + " / " + nameValuePair.getName() + "(%)",
-					DecimalUtil.formatDecimalTwo(nameValuePair.getValue() * 100) }));
+		pairList.forEach(nameValuePair -> {
+			rows.add(new String[] { "Classifications",
+					fieldName + " / " + nameValuePair.getName() + "(Value/Cards count%)",
+					DecimalUtil.formatDecimalTwo(nameValuePair.getCardCountValue() * 100) });
+			rows.add(new String[] { "Classifications",
+					fieldName + " / " + nameValuePair.getName() + "(Value/Story point%)",
+					DecimalUtil.formatDecimalTwo(nameValuePair.getStoryPointsValue() * 100) });
+		});
```

Comment:
1. `classification` duplicated code
2. There is another cost for the `getter`, like `nameValuePair.getName()`, so we can store it globally, and use the getter value,

Reply:
done